st7789/st7789.go

```diff
@@ -224,21 +253,22 @@ func (d *Device) SetRotation(rotation Rotation) {
 	madctl := uint8(0)
 	switch rotation % 4 {
 	case 0:
-		madctl = MADCTL_MX | MADCTL_MY
+		madctl = MADCTL_MX | MADCTL_MY | MADCTL_RGB
```

Comment:
are you sure MADCTL_RGB (0x00) is required here?

Reply:
Yeah, not required at all. It doesn't affect the data byte, but I thought it was more clear that MADCTL has an effect on RGB/BGR memory ordering in addition to rotation.

Reply:
Latest commit removes this change.